content/posts/2025-01-28-cli_use_cases.md

```diff
@@ -0,0 +1,269 @@
+```sh
+oc netobserv flows --enable_all --peer_ip=10.129.0.48
+```
+
+The script will connect to your cluster and start deploying eBPF agents and collector pod:
+```sh
+Checking dependencies... 
+'yq' is up to date (version v4.43.1).
+'bash' is up to date (version v5.2.26).
+Setting up... 
+cluster-admin
+creating netobserv-cli namespace
+namespace/netobserv-cli created
+creating service account
+serviceaccount/netobserv-cli created
+clusterrole.rbac.authorization.k8s.io/netobserv-cli unchanged
+clusterrolebinding.rbac.authorization.k8s.io/netobserv-cli unchanged
+creating collector service
+service/collector created
+creating flow-capture agents:
+opt: pkt_drop_enable, value: true
+opt: dns_enable, value: true
+opt: rtt_enable, value: true
+opt: network_events_enable, value: true
+opt: udn_enable, value: true
+opt: pkt_xlat_enable, value: true
+opt: filter_peer_ip, value: 10.129.0.48
+daemonset.apps/netobserv-cli created
+Waiting for daemon set "netobserv-cli" rollout to finish: 0 of 2 updated pods are available...
+Waiting for daemon set "netobserv-cli" rollout to finish: 1 of 2 updated pods are available...
+daemon set "netobserv-cli" successfully rolled out
+Running network-observability-cli get-flows... 
+pod/collector created
+pod/collector condition met
+```
+
+Once that done, it will connect to the collector and display its output:
+```sh
+------------------------------------------------------------------------
+         _  _     _       _                       ___ _    ___
+        | \| |___| |_ ___| |__ ___ ___ _ ___ __  / __| |  |_ _|
+        | .' / -_)  _/ _ \ '_ (_-</ -_) '_\ V / | (__| |__ | | 
+        |_|\_\___|\__\___/_.__/__/\___|_|  \_/   \___|____|___|
+
+------------------------------------------------------------------------
+INFO[0000] Log level: info
+Option(s): enable_all|peer_ip=10.129.0.48 
+INFO[0000] Kernel version: 5.14.0-427.50.1.el9_4.x86_64 
+INFO[0000] Starting Flow Capture...                     
+INFO[0000] Creating database...                         
+INFO[0000] flows.db created                             
+INFO[0000] Create flows table...                        
+INFO[0000] flows table created 
+```
+
+At this stage, the collector wait for incoming data. If nothing shows yet, it means that no traffic is captured. Try to open the route of your application or update the filters of the capture.
+
+Once some traffic is captured, the output will look like:
+![cli dropped]({page.image('cli/connectivity-scenario-cli-dropped.png')})
+
+You can cycle to different views using left / right arrow keys and change the displayed enrichment colomns using page up / down ones.
+Also, to adapt to your screen height, you can increase / decrease the number of displayed flows using up / down arrow keys.
+
+In this capture, we see that the traffic is blocked by OVS since it reports the `OVS_DROP_LAST_ACTION` drop cause. This probably means that a Network Policy is involved.
```

Comment:
This stuff nolonger available with network events support

Reply:
ok I can rerun this scenario and see how it goes. I'm supposed to get both drop info + event here ?

Reply:
That means this will not displayed in [overview drop cause graph](https://github.com/netobserv/network-observability-console-plugin/blob/main/web/src/utils/overview-panels.ts#L71) anymore !

We may need a new graph for events here

Reply:
we will get still pkt drops but not for ovn netpol

Reply:
I actually don't get anything in the drops info in that case
![Screenshot from 2025-02-03 11-32-07](https://github.com/user-attachments/assets/0ffb23d5-63dc-47ca-8fe4-47df2713af4e)

Whereas the events are correct:
![Screenshot from 2025-02-03 11-31-45](https://github.com/user-attachments/assets/ca4122a2-2425-4dda-be88-8dd35d416ff7)

It's really confusing in terms of UX

Reply:
https://issues.redhat.com/browse/NETOBSERV-2092

Reply:
listed the two cases in https://github.com/netobserv/netobserv.github.io/pull/6/commits/152644b7a8ba8dfe9a9ec6cd0f8b8b92cbf47626

cc @jotak we should be good to publish now :smile_cat: